avocado/utils/multipath.py

```diff
@@ -97,6 +97,24 @@ def get_mpath_name(wwid):
         return process.run(cmd, sudo=True).stdout_text.split()[0]
 
 
+def get_mapper_name(dev_mapper):
+    """
```

Comment:
can we change this method name something like `get_mpath_name_with_dm-id(dm-id)`

as `get_mapper_name` is slightly similar `get_mpath_name`.
using dm-id will help to understand quickly and people can use it.